main.css

```diff
@@ -10,14 +10,17 @@ html {
   --spacing-x-large: 4rem;
   --spacing-xx-large: 8rem;
 
-  /* font-sizes */
-  --font-x-small: 0.75rem;
-  --font-small: 0.875rem;
+  /* font-sizes
```

Comment:
better to add these as inline comments for the variables?

i.e. 
```css
--font-small: 0.75rem; /* 12px */
```

And I think we should explicitly define `font-size: 16px` at the top of the `html {}` rule

Reply:
This is what I did initially but Prettier messes up this formatting. Will add the 16px font-size.